frontend/ui/data/keyboardlayouts/ar_keyboard.lua

```diff
@@ -60,7 +60,7 @@ local alef = ar_popup.alef
 local baa = ar_popup.baa
 local jeem = ar_popup.jeem
 local daal = ar_popup.daal
-local h-aa = ar_popup.h-aa
+local haa2 = ar_popup.haa2
```

Comment:
Incidentally, underscore (`_`) is fine in a variable name.

Reply:
@Frenzie I know, but I removed them in order to isolate an issue. I'll return them in the next round of cleanups. But for now, the keyboard as is would crash KOReader for some reason.